src/App.tsx

```diff
@@ -1,42 +1,89 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
 import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
-import { UserSelector } from './components/UserSelector';
+import { UserSelector } from './components/UserSelector/UserSelector';
 import { Loader } from './components/Loader';
+import { Post, User } from './types';
+import { getPosts, getUsers } from './api/dataFromServer';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+
+  const [loadingPosts, setLoadingPosts] = useState(false);
+  const [usersPosts, setUsersPost] = useState<Post[]>([]);
+  const [loadPostError, setLoadPostError] = useState(false);
```

Comment:
If it is `loadPostError` - it must be a string
```suggestion
  const [loadPostError, setLoadPostError] = useState('');
```